### ISSUE TITLE
Please assign a version number and make a release

### ISSUE THREAD

**Issue by yurivict** [glassechidna/zxing-cpp#68]
Please consider making a release, because distros guess a version and assign it anyway, based on the Java project, see here https://repology.org/metapackage/zxing-cpp/versions

It's easier to keep track of updates when the project has a version. Without a version repology.org shows different versions that various distros assign.

**Comment by simonschmeisser** [comment]
I had a short look around and the fork/version at https://github.com/nu-book/zxing-cpp seems the most feature complete to me and they made a 1.0 release on my request. Maybe we can consolidate work there (or in a new central repository)?

**Comment by Benjamin-Dobell** [comment]
@simonschmeisser nu-book's version has had a massive overhaul, which is a Good Thing™️. However, I'm not quite sure how compatible the API is, and what platforms are supported etc.

If you could get a maintainer of that fork to chime in here with what's different that'd be helpful. If they're happy for me to do so, and it seems like a suitable replacement for most use cases, then I'll happily update the README in this repo to suggest users give it a try.

**Comment by huycn** [comment]
@Benjamin-Dobell I am one of maintainers of Nu-book's version. When we started the folk, our intention was not to have one-to-one source equivalence with original ZXing's Java code, but rather an idiomatic C++ library. As a result, it is not a drop-in replacement. However, we did keep same client-facing classes (readers, writers, binarizers, luminance sources, etc.) with their same roles. Thus, it should be quite straightforward to adapt client's code.

As for supported platforms, thanks to CMake, our code builds on Linux, Windows, Mac without issue. It also has ready-to-use wrappers to use in Android or UWP apps.

Like anyone here, we enjoy to share our work with more people, thus we'll be happy to get a link in the readme to our fork :)
